tests/test_user.py

```diff
@@ -1,4 +1,52 @@
-from app.routers.user import create_user, does_user_exist, get_users
+from datetime import datetime
+import pytest
+
+from app.routers.user import (
+    create_user, disable_user, does_user_exist, get_users
+)
+from app.database.models import User, UserEvent, Event
+
+
+# -----------------------------------------------------
+# Fixtures
+# -----------------------------------------------------
+
+
+@pytest.fixture
+def new_user(session):
+    user = create_user(
+        session=session,
+        username='new_test_username',
+        password='new_test_password',
+        email='new_test.email@gmail.com',
+        language_id='english'
+    )
+
+    return user
+
+
+# -----------------------------------------------------
+# Tests
+# -----------------------------------------------------
+
+
+def test_disabling_user(new_user, session):
```

Comment:
You should test all the possibilities :)

Reply:
Added seperate tests for the 3 different test cases :)